spec/unit/parser/functions_spec.rb

```diff
@@ -61,7 +67,7 @@ def callable_functions_from(mod)
     end
 
     it "should try to autoload the function if it doesn't exist yet" do
-      Puppet::Parser::Functions.autoloader.expects(:load)
+      Puppet::Parser::Functions.autoloader.delegatee.expects(:load)
 
       Puppet::Parser::Functions.function("name")
```

Comment:
This test also looks like it lacks an assertion

Reply:
Aside from the expectation that load is called?